### PR TITLE
Add a known issues doc.

### LINKED ISSUE #10760: v1.0.0 known issues / FAQ accumulator
This issue is intended to accumulate known errata / issues / workarounds so we can produce a useful doc upon release.  If you have a known issue that is worth putting in our 1.0 release notes, please add a comment here of the form "NEW: ..." and write a paragraph explaining the issue and the workaround or repercussions or whatever.  The core team will accumulate these into a final doc.

### PR DESCRIPTION
**Pull request by davidopp**
To save @brendandburns some time, I copied his PR and applied the review comments.

Closes #10760

@thockin @RichieEscarez @bgrant0607 @dchen1107 @saad-ali

### CODE REVIEW

File: docs/user-guide/known-issues.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      * A workaround is to ensure there is no other Ceph client using the RBD volume before mapping RBD image in read-write mode. For example, rados -p poolname listwatchers image_name.rbd can list RBD clients that are mapping the image.
+   * `exec` liveness/readiness probes leak resources due to Docker exec leaking resources (#10659)
+   * `docker load` sometimes hangs which causes the `kube-apiserver` not to start.  Restarting the Docker daemon should fix the issue (#10868)
+   * The kubelet on the master node doesn't register with the `kube-apiserver` so statistics aren't collected for master daemons (#10891)
+   * Heapster and InfluxDB both leak memory (#10653)
+   * Wrong node cpu/memory limit metrics from Heapster (https://github.com/GoogleCloudPlatform/heapster/issues/399)
+   * Services that set `type=LoadBalancer` can not use port `10250` because of Google Compute Engine firewall limitations
+   * Add on services can not be created or deleted via `kubectl` or the Kubernetes API (#11435)
```
Comment (lavalamp) [inline docs/user-guide/known-issues.md]: Addon or Add-on?

**Comment by lavalamp** [comment]
LGTM

File: docs/user-guide/known-issues.md (hunk 1 above)
Comment (bgrant0607) [inline docs/user-guide/known-issues.md]: Add-on

File: docs/user-guide/known-issues.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+## Known Issues
+
+This document summarizes known issues with existing Kubernetes releases.
+
+Please consult this document before filing new bugs.
+
+### Release 1.0.1
+
+   * If multiple Pods use the same RBD volume in read-write mode, it is possible data on the RBD volume could get corrupted. This problem has been found in environments where both apiserver and etcd rebooted and Pods were redistributed.
```
Comment (bgrant0607) [inline docs/user-guide/known-issues.md]: Not a big deal, but I would move this bullet to the bottom.

**Comment by davidopp** [comment]
Actually, please wait on merging this, I just realized we don't link to this doc from anywhere.

**Comment by davidopp** [comment]
Added link to known issues from troubleshooting guide. PTAL.

**Comment by brendandburns** [comment]
LGTM.

**Comment by lavalamp** [comment]
Did you want to change the main README, too?

**Comment by davidopp** [comment]
Do you know what the link should look like?

**Comment by davidopp** [comment]
OK, I added pointers from docs/README.md and docs/user-guide/troubleshooting.md. PTAL

**Comment by davidopp** [comment]
(sorry that should say docs/troubleshooting.md)

**Comment by lavalamp** [comment]
LGTM

File: docs/user-guide/known-issues.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      * A workaround is to ensure there is no other Ceph client using the RBD volume before mapping RBD image in read-write mode. For example, rados -p poolname listwatchers image_name.rbd can list RBD clients that are mapping the image.
+   * `exec` liveness/readiness probes leak resources due to Docker exec leaking resources (#10659)
```
Comment (ixdy) [inline docs/user-guide/known-issues.md]: it seems like the issue numbers don't auto-linkify, but maybe we don't care?

File: docs/user-guide/known-issues.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+## Known Issues
+
+This document summarizes known issues with existing Kubernetes releases.
+
+Please consult this document before filing new bugs.
+
+### Release 1.0.1
+
+   * If multiple Pods use the same RBD volume in read-write mode, it is possible data on the RBD volume could get corrupted. This problem has been found in environments where both apiserver and etcd rebooted and Pods were redistributed.
+      * A workaround is to ensure there is no other Ceph client using the RBD volume before mapping RBD image in read-write mode. For example, rados -p poolname listwatchers image_name.rbd can list RBD clients that are mapping the image.
```
Comment (ixdy) [inline docs/user-guide/known-issues.md]: rados... should be quoted in ticks.

**Comment by davidopp** [comment]
OK, I implemented the comments from @bgrant0607 which and @ixdy (except for linking the numbers)

PTAL?

File: docs/user-guide/known-issues.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+   * `exec` liveness/readiness probes leak resources due to Docker exec leaking resources (#10659)
+   * `docker load` sometimes hangs which causes the `kube-apiserver` not to start.  Restarting the Docker daemon should fix the issue (#10868)
+   * The kubelet on the master node doesn't register with the `kube-apiserver` so statistics aren't collected for master daemons (#10891)
+   * Heapster and InfluxDB both leak memory (#10653)
+   * Wrong node cpu/memory limit metrics from Heapster (https://github.com/GoogleCloudPlatform/heapster/issues/399)
+   * Services that set `type=LoadBalancer` can not use port `10250` because of Google Compute Engine firewall limitations
+   * Addon services can not be created or deleted via `kubectl` or the Kubernetes API (#11435)
+   * If a pod with a GCE PD is created and deleted in rapid sucession, it may fail to attach/mount correctly leaving PD data inaccessible (or corrputed in the worst case). (https://github.com/GoogleCloudPlatform/kubernetes/issues/11231#issuecomment-122049113)
```
Comment (ixdy) [inline docs/user-guide/known-issues.md]: two typos: sucession -> succession, corrputed -> corrupted

File: docs/user-guide/known-issues.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+### Release 1.0.1
+
+   * `exec` liveness/readiness probes leak resources due to Docker exec leaking resources (#10659)
+   * `docker load` sometimes hangs which causes the `kube-apiserver` not to start.  Restarting the Docker daemon should fix the issue (#10868)
+   * The kubelet on the master node doesn't register with the `kube-apiserver` so statistics aren't collected for master daemons (#10891)
+   * Heapster and InfluxDB both leak memory (#10653)
+   * Wrong node cpu/memory limit metrics from Heapster (https://github.com/GoogleCloudPlatform/heapster/issues/399)
+   * Services that set `type=LoadBalancer` can not use port `10250` because of Google Compute Engine firewall limitations
+   * Add-on services can not be created or deleted via `kubectl` or the Kubernetes API (#11435)
+   * If a pod with a GCE PD is created and deleted in rapid sucession, it may fail to attach/mount correctly leaving PD data inaccessible (or corrputed in the worst case). (https://github.com/GoogleCloudPlatform/kubernetes/issues/11231#issuecomment-122049113)
```
Comment (ixdy) [inline docs/user-guide/known-issues.md]: two typos: sucession -> succession, corrputed -> corrupted

**Comment by davidopp** [comment]
LGTM

(from @lavalamp :) )

**Comment by ixdy** [comment]
LGTM2

**Pull request merged**